package.json

```
@@ -13,8 +13,8 @@
     "build:assets": "gulp build",
     "build:server": "tsc && cp -r src/views dist/ && cp -r src/config dist/",
     "build": "npm run build:assets && npm run build:server",
-    "lint": "eslint 'src/*.ts' 'src/**/*.ts' 'test/src/**/*.ts'",
-    "lint:fix": "eslint 'src/*.ts' 'src/**/*.ts' 'test/src/**/*.ts' --fix",
+    "lint": "eslint 'src/*.ts' 'src/**/*.ts' 'test/**/*.ts'",
+    "lint:fix": "eslint 'src/*' 'src/**/*.ts' 'test/**/*.ts' --fix",
```

```suggestion
    "lint:fix": "eslint 'src/*' 'src/**/*.ts' 'test/**/*.ts' --fix",
```

This still needs the .ts extension as no other files should be linted.

---

Thanks corrected the typo